auction-server/src/opportunity/repository/get_live_opportunities.rs

```diff
@@ -3,19 +3,19 @@ use {
         InMemoryStore,
         Repository,
     },
-    crate::kernel::entities::PermissionKey,
+    crate::opportunity::entities,
 };
 
 impl<T: InMemoryStore> Repository<T> {
-    pub async fn get_live_opportunities_by_permission_key(
+    pub async fn get_live_opportunities(
```

Comment:
can we refactor the name of these functions. we have `get_opportunities`, `get_all_opportunities`, and `get_live_opportunities`. By just looking at the names, I can't be sure which of these is querying the db and which of the mare not.